recipe/meta.yaml

```diff
@@ -39,3 +39,4 @@ extra:
   recipe-maintainers:
     - mrocklin
     - pitrou
```

Comment:
While you are at it, could you remove me as a maintainer?

Reply:
No problem, done : )